packages/rsc-builder/buildRsc.ts

```diff
@@ -24,6 +24,9 @@ const PROJECTS: Project[] = [
     rootPath: path.join(process.cwd(), 'packages/mui-material'),
     ignorePaths: [
       'packages/mui-material/src/InitColorSchemeScript/InitColorSchemeScript.tsx', // RSC compatible
+      'packages/mui-material/src/PigmentContainer/PigmentContainer.tsx', // RSC compatible
+      'packages/mui-material/src/PigmentGrid/PigmentGrid.tsx', // RSC compatible
+      'packages/mui-material/src/PigmentStack/PigmentStack.tsx', // RSC compatible
```

Comment:
🎉

Reply:
I was wrong. The layout components still need `useDefaultProps` so `use client` is required for them.

Reply:
Based on the [discussion](https://mui-org.slack.com/archives/C061V6JNZ9Q/p1719389635626729), I remove the `useDefaultProps` for now.